pootle/apps/pootle_store/models.py

```diff
@@ -1324,7 +1324,7 @@ def real_path(self):
         return self.file.name
 
     @property
-    def is_terminology(self):
+    def is_store_terminology(self):
```

Comment:
hmm, well i thought about that, but i felt like it still implied that it was part of a terminology project, which isnt relevant